pkgs/tools/bluetooth/obex-data-server/default.nix

```diff
@@ -9,7 +9,8 @@ stdenv.mkDerivation rec {
     sha256 = "0kq940wqs9j8qjnl58d6l3zhx0jaszci356xprx23l6nvdfld6dk";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  strictDeps = true;
+  nativeBuildInputs = [ pkg-config dbus-glib ];
   buildInputs = [ libusb-compat-0_1 glib dbus-glib bluez openobex dbus ];
```

Comment:
Do we need dbug-glib in both? Can you maybe add a comment which binary is used from it?

Reply:
Thanks for the review!
dbus-glib is required in nativeBuildInputs for dbus-binding-tool. I just added a comment.